website/docusaurus.config.js

```diff
@@ -14,7 +14,7 @@ const config = {
     title: "AYON Docs",
     tagline:
         "Documentation for the most robust open-source pipeline for studios and remote teams",
-    url: "https://docs.ynput.io", // Your website URL
+    url: "https://ynput.github.io", // Your website URL
     baseUrl: "/",
```

Comment:
since we are using git hub pages, the base url must match the repository name: 
```suggestion
    baseUrl: "/ayon-documentation",
```